Ed-Fi-Alliance-OSS/Ed-Fi-AdminAPI
# [ADMINAPI-777] Enabled ReadChanges support for 5.3-cqe

On my localhost, I manually tested the following scenarios:

1. Build ODS/API from the `ODS-6093` branches. These are based on ODS/API `5.3-cqe`, with extra modifications because the actual `5.3-cqe` no longer compiles due to changes in dotnet SDK and due to some custom NuGet packages having been deleted through the normal retention policy.
   1. Run with `Using53Cqe: true` - create and update a ClaimSet with `"readChanges": true`. GET request should return the `readChanges` property.
   2.  Run with `Using53Cqe: false`  - same scenario, except the GET response should not include `readChanges`
 2. Run ODS/API v5.3 in Docker. Use of `readChanges` should cause an error, since it is not supported.
 3. Run ODS/API v6.1 in Docker. Use of `readChanges` should work fine, as in scenario 1.1 above.

Note that some integration tests are failing. This appears to be a problem with the infrastructure for supporting the `DbTests` project, rather than a problem with these changes. I will need to confirm by checking out the `main` branch and trying the tests from there.
